packages/@ember/-internals/glimmer/lib/helpers/if-unless.ts

```diff
@@ -113,20 +130,6 @@ class ConditionalHelperReference extends CachedReference {
   Hello Alex
   ```
 
-  ### Nested `if`
```

Comment:
Hmm, why delete this?

Reply:
The example doesn't really make sense with angle bracket invocation, so I thought I'd remove it.
I think we already separately document nested helpers?

Reply:
The thing I want to make sure is that we document "inline if" since that is what this section was documenting. If we can confirm it is still documented, then 👍 	seems good to delete this. Otherwise we need to bring it back...